app/services/unverified-reports.js

```diff
@@ -0,0 +1,60 @@
+import { computed } from "@ember/object";
+import Service, { inject as service } from "@ember/service";
+import moment from "moment";
+
+const INTERVAL_DELAY = 10 * 60000; // 10 Minutes
```

Comment:
I usually go from the smallest unit to what I need. In this case:
`1000 * 60 * 10`

But this is just my personal preference. No need to change this!